src/mozilla_sec_eia/models/sec10k/ex_21/train_extractor.py

```diff
@@ -134,23 +133,20 @@ def load_test_train_set(
     return split_dataset["train"], split_dataset["test"]
 
 
-def train_model(
-    labeled_json_path: str,
-    gcs_training_data_dir: str = "labeled",
-    model_output_dir="layoutlm_trainer",
-    test_size=0.2,
-):
-    """Train LayoutLM model with labeled data.
+class FineTuneConfig(Config):
```

Comment:
non-blocking: Fine tuning is basically taking stock LayoutLM and then training it with our labeled data, so it knows what we're looking for?